### PR TITLE
Scenario dashboard redesign

### PR DESCRIPTION
**Pull request by Enoch-Hsiao**
If scenario is unfinished, I show the buttons "Edit," "Delete," "Share." If scenario is finished, I add the extra button "Data." 
Dashboardui

### CODE REVIEW

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/ScenarioCard.js [hunk 1]
```diff
@@ -40,11 +40,167 @@ const useStyles = makeStyles((theme) => ({
...
-  const data = props.data;
-  
+  const { data } = props;
+  const { id, scenarioName, className, finished } = data;
+
+  //If scenario is unfinished, we show the buttons "Edit," "Delete," "Share"
+  //If scenario is finished, we show the button "Edit," "Delete," "Share," "View Student Data"
+  const buttons = 
+    finished ? 
+      <Grid
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/ScenarioCard.js]: Very longwinded solution to figuring out which buttons to show. If anyone has any ways to make this implementation cleaner, would be greatly appreciated.

Comment (powellsl99) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/ScenarioCard.js]: Why not just use the ternary for the bottom row? Thats all thats changing. Would reduce a lot of the copy paste

Comment (Enoch-Hsiao) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/ScenarioCard.js]: Got it.

**Review by powellsl99 — APPROVED** [review]
Looks good!

**Pull request merged**